tools/samurai/Makefile

```diff
@@ -0,0 +1,26 @@
+# 
+# Copyright (C) 2013 OpenWrt.org
```

Comment:
2013? I guess this could also be added as SPDX thing.

Reply:
Copied from some other Makefile. I'll remove.

Reply:
Yes, please use SPDX identifiers on all new files.

Reply:
Done.